lib/src/main/java/com/diffplug/spotless/generic/ReplaceRegexStep.java

```diff
@@ -49,5 +58,14 @@ private static final class State implements Serializable {
 		FormatterFunc toFormatter() {
 			return raw -> regex.matcher(raw).replaceAll(replacement);
 		}
+
+		FormatterFunc toFormatFinder() {
+			return raw -> {
+				if (regex.matcher(raw).find()) {
+					throw new IllegalArgumentException(replacement);
+				}
+				return raw;
+			};
+		}
```

Comment:
This is a perfect place to use our new Lint system.

https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md#lints

That will allow Spotless to flag multiple wildcard imports at once rather than only one at a time.